docs/machine-learning/tutorials/health-violation-classification-model-builder.md

```diff
@@ -0,0 +1,169 @@
+
+The time required for the model to train is proportional to the amount of data. Model Builder automatically selects a default value for **Time to train (seconds)** based on the size of your data source.
+
+1. Although Model Builder sets the value of **Time to train (seconds)** to 10 seconds, increase it to 30 seconds. Training for a longer period of time allows Model Builder to explore a larger number of algorithms and combination of parameters in search of the best model.
+1. Select **Start Training**.
+
+    Throughout the training process, progress data is displayed in the `Progress` section of the train step.
+
+    - Status displays the completion status of the training process.
+    - Best accuracy displays the accuracy of the best performing model found by Model Builder so far. Higher accuracy means the model predicted more correctly on test data.
+    - Best algorithm displays the name of the best performing algorithm performed found by Model Builder so far.
+    - Last algorithm displays the name of the algorithm most recently used by Model Builder to train the model.
+
+1. Once training is complete, select the **evaluate** link to move to the next step.
+
+## Evaluate the model
+
+The result of the training step will be one model, which had the best performance. In the evaluate step of the Model Builder tool, the output section, will contain the algorithm used by the best performing model in the **Best Model** entry along with metrics in **Best Model Accuracy**. Additionally, a summary table containing top five models and their metrics.
+
+If you're not satisfied with your accuracy metrics, some easy ways to try to improve model accuracy are to increase the amount of time to train the model or use more data. Otherwise, select the **code** link to move to the final step in the Model Builder tool.
+
+## Add the code to make predictions
+
+Two projects will be created as a result of the training process.
+
+- RestaurantViolationsML.ConsoleApp: A C# .NET Core Console application that contains the model training and sample consumption code.
+- RestaurantViolationsML.Model: A .NET Standard class library containing the data models that define the schema of input and output model data, the saved version of the best performing model during training and a helper class called `ConsumeModel` to make predictions.
+
+1. In the code step of the Model Builder tool, select **Add Projects** to add the autogenerated projects to the solution.
+1. Open the *Program.cs* file in the *RestaurantViolations* project.
+1. Add the following using statement to reference the *RestaurantViolationsML.Model* project:
+
+    ```csharp
+    using System;
+    using RestaurantViolationsML.Model;
+    ```
+
+1. To make a prediction on new data using the model, create a new instance of the `ModelInput` class inside the `Main` method of your application. Notice that the fare amount is not part of the input. This is because the model will generate the prediction for it. 
```

Comment:
```suggestion
1. To make a prediction on new data using the model, create a new instance of the `ModelInput` class inside the `Main` method of your application. Notice that the fare amount is not part of the input. This is because the model generates the prediction for it. 
```